src/build/__main__.py

```diff
@@ -117,10 +120,10 @@ def _build_in_isolated_env(
 
 
 def _build_in_current_env(
-    srcdir: PathType,
-    outdir: PathType,
-    distribution: str,
-    config_settings: ConfigSettingsType | None,
+    srcdir: _types.StrPath,
```

Comment:
I'd prefer to import these directly without needing to namespace access it like here...

Reply:
Agreed, done.